protocol/activity_center_persistence.go

```diff
@@ -980,32 +1085,89 @@ func (db sqlitePersistence) AcceptActivityCenterNotificationsForInvitesFromUser(
 		return nil, err
 	}
 
-	return notifications, nil
+	return notifications, updateActivityCenterState(tx, updatedAt)
 }
 
 func (db sqlitePersistence) MarkAllActivityCenterNotificationsRead(updatedAt uint64) error {
-	_, err := db.db.Exec(`UPDATE activity_center_notifications SET read = 1, updated_at = ? WHERE NOT read AND NOT deleted`, updatedAt)
+	var tx *sql.Tx
+	var err error
+
+	tx, err = db.db.BeginTx(context.Background(), &sql.TxOptions{})
```

Comment:
Can't we use `:=` here?

```suggestion
	tx, err := db.db.BeginTx(context.Background(), &sql.TxOptions{})
```